src/materials/Material.js

```diff
@@ -249,7 +249,7 @@ Material.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 		if ( this.blending !== NormalBlending ) data.blending = this.blending;
 		if ( this.flatShading === true ) data.flatShading = this.flatShading;
 		if ( this.side !== FrontSide ) data.side = this.side;
-		if ( this.vertexColors !== NoColors ) data.vertexColors = this.vertexColors;
+		if ( this.vertexColors === true || this.vertexColors > 0 ) data.vertexColors = true;
```

Comment:
Do we need ` || this.vertexColors > 0` here?

Reply:
I think if we change the check to just `vertexColors`, it should be also work. Let me make this change with a new commit 👍